packages/@okta/vuepress-theme-prose/global-components/EmbeddedBrowserWarning.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <blockquote>
+    <p><strong>Note</strong>: We strongly advise against using WebViews for authentication on mobile apps because of major security implications. See <a href="https://datatracker.ietf.org/doc/html/draft-ietf-oauth-native-apps">OAuth 2.0 for Native Apps</a> for more information. You should use our native SDKs instead.</p>
```

Comment:
```suggestion
    <p><strong>Note</strong>: We strongly advise against using WebViews for authentication on mobile apps because of major security implications. See <a href="https://datatracker.ietf.org/doc/html/draft-ietf-oauth-native-apps">OAuth 2.0 for Native Apps</a>. You should use our native SDKs instead.</p>
```